app/adapters/stripe-auth.js

```diff
@@ -0,0 +1,39 @@
+import ApplicationAdapter from './application';
+import Ember from 'ember';
+
+const {
+  get
+} = Ember;
+
+export default ApplicationAdapter.extend({
+  urlForQueryRecord(query) {
+    query = query || {};
```

Comment:
Rather than using `|| {}` what do you think about setting a default value in the function declaration?

`urlForQueryRecord(query = {}) { ... }`